client/components/data/query-dotorg-plugins/index.jsx

```diff
@@ -0,0 +1,43 @@
+import config from '@automattic/calypso-config';
+import PropTypes from 'prop-types';
+import { useEffect, useRef } from 'react';
+import { useDispatch, useSelector } from 'react-redux';
+import { useInterval } from 'calypso/lib/interval';
+import { fetchPluginData as wporgFetchPluginData } from 'calypso/state/plugins/wporg/actions';
+import { getAllPlugins } from 'calypso/state/plugins/wporg/selectors';
+
+const PLUGIN_RETRIEVE_INTERVAL = 500;
+const BATCH_PLUGIN_RETRIEVE_COUNT = 5;
+
+function QueryDotorgPlugins( { pluginSlugList } ) {
+	const dispatch = useDispatch();
+	const queueRef = useRef( [] );
+
+	const dotorgPlugins = useSelector( ( state ) => getAllPlugins( state ) );
+
+	useEffect( () => {
+		pluginSlugList.forEach( ( pluginSlug ) => {
+			if ( ! queueRef.current.includes( pluginSlug ) && ! dotorgPlugins[ pluginSlug ] ) {
+				queueRef.current.push( pluginSlug );
+			}
+		} );
+	}, [ pluginSlugList, dotorgPlugins ] );
+
+	useInterval( async () => {
+		if ( pluginSlugList.length && config.isEnabled( 'bulk-plugin-management' ) ) {
```

Comment:
Lets just move this into a variable below `dotorgPlugins` so that this method is not called every interval tick.

```js
const isBulkManagementEnabled = config.isEnabled('bulk-plugin-management');
```

Reply:
We good to go after this comment is resolve

Reply:
Nice. So I'll deploy as soon as the tests pass, thanks!